docs/content/posts/release-ml-2.0.0.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post 
+title:  "Apache Flink ML 2.0.0 Release Announcement"
+date: 2022-01-07T08:00:00.000Z
+type: docs
+categories: news
+authors:
+- lindong:
+  name: "Dong Lin"
+- gaoyun:
+  name: "Yun Gao"
+
+excerpt: The Apache Flink community is excited to announce the release of Flink ML 2.0.0! This release involves a major refactor of the earlier Flink ML library and introduces major features that extend the Flink ML API and the iteration runtime, such as supporting stages with multi-input multi-output, graph-based stage composition, and a new stream-batch unified iteration library.
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+The Apache Flink community is excited to announce the release of Flink ML
```

Comment:
Could we avoid copy/paste the entire blog post from flink-web? We can use URL if needed.

Reply:
According to offline discussions, I would remove the blog post section from this PR. Blogs about Flink ML can be placed directly in flink-web repo.